lunatrace/bsl/ingest-worker/cmd/ingestworker/vulnerability/vulnerability.go

```diff
@@ -62,22 +65,52 @@ func NewCommand(p Params) clifx.CommandResult {
 						sourceRelativePath := ctx.String("source-relative-path")
 						cron := ctx.String("cron")
 
-						log.Info().
-							Msg("Updating CWEs")
-						err := p.CWEIngester.Ingest(ctx.Context)
-						if err == nil {
+						runIngestion := func() error {
+							log.Info().
+								Msg("Updating CWEs")
+
+							err := p.CWEIngester.Ingest(ctx.Context)
+
+							if err != nil {
```

Comment:
Please collect errors instead of discarding them. Returning an error from this func should cause the tool to exit with nonzero exit code which will trigger monitoring if it fails.